components/console-backend-service/internal/gqlschema/schema.graphql

```diff
@@ -406,6 +406,18 @@ type LimitRangeItem {
     defaultRequest: ResourceType!
 }
 
+input LimitRangeInput {
+    default: ResourceTypeInput!,
+    defaultRequest: ResourceTypeInput!
+    max: ResourceTypeInput!
+    type: String!
+}
+
+input ResourceTypeInput {
```

Comment:
There is already a type for that: [`ResourceValuesInput `](https://github.com/akucharska/kyma/blob/eedde0c38f6ef7efe96f916592684681511c1533/components/console-backend-service/internal/gqlschema/schema.graphql#L583)

Reply:
`ResourceTypeInput` removed, changed to `ResourceValuesInput`